src/richie/apps/courses/templates/courses/plugins/_course_plugin.scss

```diff
@@ -0,0 +1,15 @@
+.course-plugin-container {
+  @extend .course-glimpse-container;
+  display: inline-block;
+  min-width: 250px;
+  margin: auto;
+}
+
+.course-plugin {
+  @extend .course-glimpse;
+  max-width: 100%;
+  
+  &__link {
```

Comment:
Sorry about this, but I think a tenet of BEM is that a modifier class (such as `.course-glimpse--link`) is supposed to be used on the same element as the class it's modifying (`.course-glimpse`).

It may work in this instance but it feels wrong to use them as parent & child, even through an extend.

Reply:
This is what I'm talking about:

<img width="366" alt="capture d ecran 2018-11-13 a 17 35 52" src="https://user-images.githubusercontent.com/1932937/48428129-9e1c3b00-e76a-11e8-9014-ba1dfefdd228.png">

`.course-glimpse` & `.course-glimpse--link` need to be **on the same element**, even if you're just applying them through an extend.

I guess what I'm thinking this should look like is more something like this:

```
.course-plugin {
  @extend .course-glimpse;
  @extend .course-glimpse--link;
  max-width: 100%;
}
```

I think this matches the way they were meant to be used, as can be seen eg. on the organization detail page https://github.com/openfun/richie/blob/master/src/richie/apps/courses/templates/courses/cms/organization_detail.html#L34.

Sorry if my previous comments were a little bit unclear 😅

Reply:
Oh i clearly misunderstood the example then... Give me a moment, it's gonna be clean in a moment.